ckanext/dcatapchharvest/profiles.py

```diff
@@ -229,6 +229,18 @@ def _clean_datetime(self, datetime_value):
         except (ValueError, KeyError, TypeError, IndexError):
             return None
 
+    def _get_eu_ccrualPeriodicity(self, subject, predicate):
```

Comment:
```suggestion
    def _get_eu_accrual_periodicity(self, subject, predicate):
```
(typo + Python style)

Reply:
you are right, it should be a snake_case